pytest.ini

```diff
@@ -50,11 +50,13 @@ pep8ignore =
     matplotlib/testing/jpl_units/UnitDbl.py E201 E202 E203
     matplotlib/testing/jpl_units/UnitDblConverter.py E201 E202 E203 E251 E302 E501 E711
     matplotlib/testing/jpl_units/UnitDblFormatter.py E201 E202 E251 E302
+    matplotlib/tests/test_contour.py E501
```

Comment:
The PEP8 exceptions list should only get smaller, not bigger. Probably will be fixed by using the `py.test` method noted by @WeatherGod.